bundles/page/versioning.rst

```diff
@@ -8,7 +8,7 @@ restored, which will result in a new draft.
 
 .. note::
 
-   Mind that this feature is only available for Jackrabbit, because Jackalope does
+   Mind that this feature is only available for :doc:`../../cookbook/jackrabbit`, because Jackalope does
```

Comment:
This will write "for Using Jackrabbit", because that's the title of this page....

Reply:
fixed